tests/test_time_series/test_forecasting/test_expanding_window_features.py

```diff
@@ -550,3 +550,20 @@ def test_transform_x_y(df_time):
     assert len(Xt) == len(yt)
     assert len(y) != len(yt)
     assert (Xt.index == yt.index).all()
+
+
+def test_error_duplicate_functions(df_time):
+    with pytest.raises(ValueError):
+        ExpandingWindowFeatures(
+            variables=["ambient_temp"], functions=["sum", "sum"], periods=2, freq="15T"
+        )
+
+
+def test_error_native_functions(df_time):
```

Comment:
which error is this testing?

Reply:
This one (deactivated the test and took a screenshot from coverage)
<img width="872" alt="Bildschirmfoto 2024-07-03 um 16 41 25" src="https://github.com/feature-engine/feature_engine/assets/119943720/f491933c-472a-417c-a998-a5c92c3f278c">

Reply:
thank you! In that case, we need to use parametrize and test more than 1 incorrect value, and add an assert for the error message.